components/Article.jsx

```diff
@@ -0,0 +1,37 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const Article = () => {
+    const {article_id} = useParams()
+    const [article, setArticle] = useState([])
+    const [loading, setLoading] = useState(true)
+    useEffect(() => {
+        axios.get(`https://newssite-zy4v.onrender.com/api/articles/${article_id}`).then((res) => {
```

Comment:
Please put your axios requests in a separate file and import the function for cleanliness.